posts/jhipster-deepdive-part1.md

```diff
@@ -0,0 +1,51 @@
+# JHipster Deep Dive, Part One
```

Comment:
Can you add a metadata block? Look at other posts, e.g. https://github.com/ippontech/blog-usa/edit/master/posts/10-tips-and-tricks-for-cassandra.md

The title will be included in this block, so you will need to remove one `#` from your titles (e.g. `Getting started` will be `#`).